src/ReactDataGrid.js

```diff
@@ -857,7 +857,7 @@ const ReactDataGrid = React.createClass({
   },
 
   dragEnabled: function(): boolean {
-    return this.props.onGridRowsUpdated !== undefined || this.props.onCellsDragged !== undefined;
+    return (this.props.onGridRowsUpdated !== undefined || this.props.onCellsDragged !== undefined) && this.props.enableDragAndDrop !== false;
```

Comment:
Can we check `enableDragAndDrop` first and only then check the callbacks? I guess most times it won't need to check the callbacks, checking if the drag and drop is enable first sounds like a reasonable thing to do.

Aside from that, having a falsy check in here is safe and would make the code much more readable.

Would end up being

```
return this.props.enableDragAndDrop && (this.props.onGridRowsUpdated || this.props.onCellsDragged);

```